src/Markup/Avalonia.Markup.Xaml/Templates/Template.cs

```diff
@@ -4,14 +4,14 @@
 
 namespace Avalonia.Markup.Xaml.Templates
 {
-    public class Template : ITemplate<Control>
+    public class Template : ITemplate<Control?>
```

Comment:
Not sure how much sense a `Template` that returns null is, though I guess it is can't hurt in this case.

Reply:
Apparently it's been added in #5725 for consistency with WPF.

`TemplateContent.Load` used a few lines below can definitely return `null` since that PR and the parent type `ITemplate<TControl>` was already annotated with `where TControl : Control?`.